### PR TITLE
Bug/1364 inform the user why cannot run on hadoop

### LINKED ISSUE #1364: Inform the user why the datastore in not valid for running on Hadoop
- The button for running a job on Hadoop greyed out even if the job contains a hdfs resource. 
Inform the user why the CSV datastore is not valid. 

- [ ]  Multilines  are allowed
- [ ] The encoding is not UTF-8

### PR DESCRIPTION
**Pull request by ClaudiaPHI**
Fixes #1364

### CODE REVIEW

File: engine/env/spark/src/main/java/org/datacleaner/spark/utils/HadoopJobExecutionUtils.java [hunk 1]
```diff
@@ -53,26 +53,28 @@
...
+                final CsvDatastore csvDatastore = (CsvDatastore) datastore;
+                if (!isValidMultilines(csvDatastore) || !isValidEnconding(csvDatastore)){
+                    return false; 
+                }
+            }
+            return true; 
+        }else{
+            return false; 
+        }
+    }
```
Comment (LosD) [inline engine/env/spark/src/main/java/org/datacleaner/spark/utils/HadoopJobExecutionUtils.java]: There seems to be some code style issues here (spaces around scopes etc.)

**Comment by LosD** [comment]
I don't see the message anywhere in the code? Did that already exist?

**Comment by ClaudiaPHI** [comment]
The messages are in commercial edition.

**Comment by LosD** [comment]
Ah, super.

Except for the formatting in a single method, code LGTM.

Didn't test yet, though.

**Pull request merged**